index.html

```diff
@@ -26,6 +26,159 @@ <h1 class="career">I'm a software developer</h1>
                 <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
             </ul>
         </section>
+        <section>
+            <a class="works"><img src="Backgroud-images/Disabled.png" alt="works-button"></a>
+            <h1 class="recent-works">My Recent Works</h1>
+            <img class="separation-bar" src="Backgroud-images/Indicator.png" alt="separation-bar">
+        </section>
+        <section class="container">
```

Comment:
It's crucial that our projects have the same visual presentation as Figma's design to have a clear differentiation for the user 👀 Kindly perform the necessary changes in your code so that your cards follow the design 🙌

Yours
![image](https://user-images.githubusercontent.com/15807343/153529212-7d9f4d7e-0e45-48f7-a240-b25ccecfec9f.png)

Figma's 
![Captura](https://user-images.githubusercontent.com/15807343/153529057-34578877-00d7-49f4-85e6-dd55ea68cdf7.PNG)